plantuml-parser-core/src/test/java/com/shuzijun/plantumlparser/core/TestRecordAndDefaultConstructor.java

```diff
@@ -0,0 +1,136 @@
+package com.shuzijun.plantumlparser.core;
+
+import com.github.javaparser.ParserConfiguration;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.Scanner;
+
+public class TestRecordAndDefaultConstructor {
+
+    private static ParserConfig makeParserConfig(String filename, boolean showDefaultCon) {
+        ParserConfig parserConfig = new ParserConfig();
+        parserConfig.setLanguageLevel(ParserConfiguration.LanguageLevel.JAVA_17);
+        parserConfig.setShowConstructors(true);
+        parserConfig.setShowDefaultConstructors(showDefaultCon);
+        parserConfig.addFieldModifier("private");
+        parserConfig.addMethodModifier("public");
+        parserConfig.addMethodModifier("default");
+        File file = new File(Objects.requireNonNull(
+            TestDefaultConstructor.class.getClassLoader().getResource(filename)).getFile());
+        parserConfig.addFilePath(file.getAbsolutePath());
+        return parserConfig;
+    }
+
+    private static String getUml(String filename, boolean showDefaultCon) throws IOException {
+        ParserProgram parserProgram = new ParserProgram(makeParserConfig(filename, showDefaultCon));
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        PrintStream oldOut = System.out;
+        System.setOut(new PrintStream(baos));
+        parserProgram.execute();
+        String result = baos.toString();
+        System.setOut(oldOut);
+        return result;
+    }
+
+    private static boolean isUmlEquivalent(String uml1, String uml2, String umlFilename)
```

Comment:
The meaning of the 3 parameters isn't clear. My guess is that uml1 is compared to either uml2 or what is read from umlFilename. If so, those should be two separate methods.